packages/contracts/deploy/migrations/migrate.ts

```diff
@@ -100,7 +89,6 @@ export const migrator = {
                     gas: addTokenGasEstimate,
                 },
             );
-        });
-        await Promise.all([...addDummyTokenPromises, ...addTokenPromises]);
+        }
     },
 };
```

Comment:
Let's leave a comment at the top of this file explaining why everything is sync. It is non-obvious.